tree/tree/inc/TTreeCacheUnzip.h

```diff
@@ -29,26 +29,41 @@
 
 #include <queue>
 
+#ifdef R__USE_IMT
+#include "tbb/task.h"
+#include "tbb/queuing_rw_mutex.h"
+#include <atomic>
+#include <mutex>
+#include <thread>
+#include <vector>
+#endif
+
 class TTree;
 class TBranch;
 class TThread;
 class TCondition;
 class TBasket;
 class TMutex;
 
+class MappingTask;
+class UnzipTask;
+
 class TTreeCacheUnzip : public TTreeCache {
+
 public:
    // We have three possibilities for the unzipping mode:
    // enable, disable and force
    enum EParUnzipMode { kEnable, kDisable, kForce };
 
 protected:
 
+   friend class MappingTask;
+   friend class UnzipTask;
+   tbb::task *root;
```

Comment:
In general, there's a huge number of complaints from `clang-format`.  @zzxuanyuan can you please take the patch generated by Travis-CI and apply it?  It'll take care of these odds-n-ends so human reviewers don't have to bother with them.